### DISCUSSION TITLE
pane frames toggle not working for floating windows

### DISCUSSION THREAD

**Discussion by mjkpolo** [zellij-org/zellij#3666] [Q&A] [answered]
Do you know of any way to disable pane frames for floating windows as well? Toggling with `ctrl + p + z` is shown in pictures below.

![1](https://github.com/user-attachments/assets/7e04dfc0-ff09-4ed7-bfdb-54a91125d29c)
![2](https://github.com/user-attachments/assets/d7b85ccb-fd6d-42b6-93d3-19c94c0bd707)

**Comment by imsnif** [comment, marked as answer]
To be honest, this was a stylistic choice. I didn't think floating panes would look very good without a frame. I think one wouldn't be able to recognize where the floating pane ends and the pane under it begins.

**Reply by mjkpolo** [reply]
that's very true... I guess I just don't like the title bar but you're right you totally need the border
